src/http/verification.rs

```diff
@@ -0,0 +1,39 @@
+use html_parser::Dom;
+use std::str::FromStr;
+use url::Url;
+
+pub async fn is_verified(input_url: Url, expected_url: Url) -> bool {
```

Comment:
Docstring along the lines of "Verifies the identity of an HTTPS link URL if we find a `rel=me` link pointing to `expected_url` on the `input_url."